pkg/visor/init.go

```diff
@@ -344,6 +345,59 @@ func initDmsgCtrl(ctx context.Context, v *Visor, _ *logging.Logger) error {
 	return nil
 }
 
+func initDmsgHTTPLogServer(ctx context.Context, v *Visor, log *logging.Logger) error {
+	dmsgC := v.dmsgC
+	if dmsgC == nil {
+		return nil
+	}
+	var dmsgTimeout = v.conf.LogRotationInterval
+	logger := v.MasterLogger().PackageLogger("dmsghttp_logserver").WithField("timeout", dmsgTimeout)
```

Comment:
Is there any value for us to log this info?

Reply:
I think it would be good to indicate what folder are we serving and will also be an indication that we are serving over dmsghttp successfully.